outputs.tf

```
@@ -0,0 +1,35 @@
+output "server_id" {
+  description = "The ID of this MySQL server."
+  value       = azurerm_mysql_flexible_server.this.id
+}
+
+output "server_name" {
+  description = "The name of this MySQL server."
+  value       = azurerm_mysql_flexible_server.this.name
+}
+
+output "server_fqdn" {
+  description = "The fully qualified domain name (FQDN) of this MySQL server."
+  value       = azurerm_mysql_flexible_server.this.fqdn
+}
+
+output "administrator_login" {
+  description = "The login username of the administrator for this MySQL server."
+  value       = azurerm_mysql_flexible_server.this.administrator_login
+}
+
+output "administrator_password" {
+  description = "The login password of the administrator for this MySQL server."
+  value       = azurerm_mysql_flexible_server.this.administrator_password
+}
+
+output "databases" {
+  description = "A map of MySQL databases."
+  value = {
+    for database in azurerm_mysql_flexible_database.this :
+    database.name => {
+      name = database.name
+      id   = database.id
+    }
+  }
+}
```

Should be split into multiple outputs of simpler types ("list of string" is simpler than "list of object of any"), similar to [network module](https://github.com/equinor/terraform-azurerm-network/blob/3433c5dd513f040e339f63c4706ce7e1cad9cc37/outputs.tf#L30-L42)

```suggestion
output "database_names" {
  description = "A map of database names."
  value = {
    for k, v in azurerm_mysql_flexible_database.this : k => v.name
  }
}

output "database_ids" {
  description = "A map of database IDs."
  value = {
    for k, v in azurerm_mysql_flexible_database.this : k => v.id
  }
}
```